docs/source/examplecleanup.rst

```diff
@@ -6,55 +6,87 @@ End-to-End Example
 
 An End-to-End example walkthrough of a tartufo scan and the process of purging the dirty evil passwords that somehow ended up in your code commits. We will use an additional tool: ``BFG`` (https://rtyley.github.io/bfg-repo-cleaner/, more on this later!). 
 
-#. Use ``tartufo`` to scan your repository and find any secrets in its history!
 
-   Use what you've learned to scan your repo!
+#. OPTIONAL Development only: Setup poetry if you want to use the most recent non-released build from github (may not be stable)
+
+   This project uses [Poetry] to manage its dependencies and do a lot of the heavy lifting. So you'll need to clone the tartufo repo and setup poetry!
+
+
+   .. code-block:: console
+
+      git clone git@github.com:godaddy/tartufo.git
+      
+   Development Use Only Poetry Setup: https://tartufo.readthedocs.io/en/master/CONTRIBUTING.html#setting-up-a-development-environment
```

Comment:
Instead of an absolute link to the contributing doc, you can use a Sphinx ref here. I believe what you will want here is

```rst
:doc:`CONTRIBUTING#setting-up-a-development-environment`
```
This makes sure the link works in all environments, across all build hosts, and continues working if the doc URL ever changes.

Reply:
yes, that is way better, thank you!

Reply:
oh, but will that reference master?

Reply:
Using that syntax will automatically reference whatever version is being built/viewed. So if you're viewing the `master` version of the docs, it will point there, if you're viewing the `latest` version of the docs it will point there, etc.

And once we switch 2.0 over to be our `stable` version, the same will apply there as well. :)